src/price/ProgressivePriceThreshold.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+declare(strict_types=1);
+
+namespace hiqdev\php\billing\price;
+
+use hiqdev\php\units\Unit;
+use Money\Money;
+use hiqdev\php\units\Quantity;
+
```

Comment:
We've put here some specific ideas, such as storing scalars instead of objects. Elaborate this decision in PHPDoc please.